setup.py

```diff
@@ -23,16 +23,15 @@ def tuple_list(directory):
     README = ''
 
 # Copy all data_files from the 'static'- and 'templates'-folder into their destination
-data_files_list = tuple_list('static')
-data_files_list.extend(tuple_list('templates'))
-# TODO: also add config-file to data_files_list
+data_files_list = tuple_list('flask-dashboard/static')
+data_files_list.extend(tuple_list('flask-dashboard/templates'))
```

Comment:
i looked a lot for the solution too, but it's too lines of manifest file, as the first link shows. it would be nice if you could try to replace the many lines with the few.